templates/utils.js

```diff
@@ -23,3 +25,5 @@ export const withDefaults =
 
     return template(argsWithDefault);
   };
+
+export const deepStringify = val => inspect(val, { depth: null, compact: true, maxArrayLength: null, maxStringLength: null })
```

Comment:
NIT: Instead of calling `deepStringify` should we just call `stringify`?

Reply:
renamed + added a note about importing [6625060](https://github.com/scaffold-eth/create-eth/pull/135/commits/66250607bdea2caafa379412939b2e2fcbe7938a)